safe/gui/tools/wizard_dialog.py

```diff
@@ -1980,34 +1979,13 @@ def on_tblFunctions1_cellDoubleClicked(self, row, column):
            executed when the category selection changes.
         """
         self.pbnNext.click()
-
     # pylint: enable=W0613
-    # noinspection PyPep8Naming
-    def on_rbHazSingle_toggled(self):
-        """Reload the functions table
-
-        .. note:: This is an automatic Qt slot
-           executed when the radiobutton is activated.
-        """
-        if self.rbHazSingle.isChecked():
-            self.populate_function_table_1()
-
-    # noinspection PyPep8Naming
-    def on_rbHazMulti_toggled(self):
-        """Reload the functions table
-
-        .. note:: This is an automatic Qt slot
-           executed when the radiobutton is activated.
-        """
-        if self.rbHazMulti.isChecked():
-            self.populate_function_table_1()
 
     def populate_function_table_1(self):
         """Populate the tblFunctions1 table with available functions."""
-        if self.rbHazSingle.isChecked():
-            hazard_category = hazard_category_single_event
-        else:
-            hazard_category = hazard_category_multiple_event
+        # The hazard category radio buttons are now removed -
+        # make this parameter of IFM.available_hazards() optional
```

Comment:
As we discussed in gitter, I will leave it just like this for now. I have updated all IF to support single and multiple events. So, no need to update the IFM (for now). We can do it later if needed.

Btw, LGTM.